helsingborg-stad/app-mitt-helsingborg
# feat(encryption): force password-based encryption

BREAKING CHANGE: breaks existing solo-applicant cases

## Explain the changes you’ve made

Remove device-local aes strategy and enforce password-based encryption on all cases.

Bonus: solo applicant can now "recover" cases by entering pin (if they remember it/have the previous device available), however texts are not updated for this.

## Explain why these changes are made

To simplify encryption and cause new-application encryption with co-applicant to become a non-issue.

## How to test

Concrete example:

1. Checkout this branch
2. Build etc.
3. Verify encryption still works as usual from an UX-perspective.
4. ...etc.

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Other notes

Requires API update (https://github.com/helsingborg-stad/helsingborg-io-sls-api/pull/447) to work.
